com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -563,6 +567,35 @@ public int additionalLightsShadowmapResolution
             get { return (int)m_AdditionalLightsShadowmapResolution; }
         }
 
+        public int additionalLightsShadowResolutionTierLow
+        {
+            get { return (int)m_AdditionalLightsShadowResolutionTierLow; }
+        }
+
+        public int additionalLightsShadowResolutionTierMedium
+        {
+            get { return (int)m_AdditionalLightsShadowResolutionTierMedium; }
+        }
+
+        public int additionalLightsShadowResolutionTierHigh
+        {
+            get { return (int)m_AdditionalLightsShadowResolutionTierHigh; }
+        }
```

Comment:
Since these are public they should probably be documented?

Reply:
Thanks for the suggestion, I added that in 06f2e77.